src/js/node/stream.ts

```diff
@@ -5832,6 +5914,21 @@ function getNativeReadableStream(Readable, stream, options) {
   return new NativeReadable(ptr, options);
 }
 
+// Cargo culted from the above function
+// TODO: might not be correct impl
+function getNativeWritableStream(stream, options) {
```

Comment:
This doesn't refer to a web `WritableStream`, so let's make sure to not name it in a way that makes it sound like it does
```suggestion
function getNativeWritable(stream, options) {
```